x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -2,6 +2,10 @@
   height: 100%;
 }
 
+.lnsTableCell--multiline {
+  white-space: pre;
```

Comment:
I believe this needs to be changed to `pre-wrap`. Doing so will allow line breaks at the end of element containing the text (rather than overflowing it, as shown in screenshot below).

```suggestion
  white-space: pre-wrap;
```

![image](https://user-images.githubusercontent.com/3884767/173905165-b778d5a1-d977-40f9-b5fb-f0d6848a62fd.png)